### PR TITLE
Update user profile

### PR DESCRIPTION
**Pull request by armandoroman1016**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [X] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by BDesselle — APPROVED** [review]

**Review by agordon1123 — APPROVED** [review]
🔥

**Review by Artyk77-zz — APPROVED** [review]

File: mission-control-client/src/App.js [hunk 1]
```diff
@@ -6,20 +6,28 @@ import {
...
+// components
 import Layout from "./components/layout/Layout";
-import "./styles/index.scss";
 import Registration from "./components/auth/Registration.js";
 import Login from "./components/auth/Login";
 import ProjectMore from "./components/dashboard/admin-dashboard/ProjectMore";
 import Bad from "./components/layout/Bad";
 import AdminDash from "./components/dashboard/admin-dashboard/DashboardHome";
 import UserDash from "./components/dashboard/user-dashboard/DashboardHome";
+import EditProfile from "./components/user_settings/EditProfile";
```
Comment (dionys93) [inline mission-control-client/src/App.js:18]: From `user_settings` to `user-settings`

File: mission-control-client/src/components/user_settings/EditForm.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            email: email || '',
+            newemail: newemail || '',
+            confirmemail: confirmemail || '',
+            password: password || '',
+            newpassword: newpassword || '',
+            confirmpassword: confirmpassword || ''
+        }    
+    },
+    validationSchema: Yup.object().shape({
+        // password: Yup.string().required()
```
Comment (dionys93) [inline mission-control-client/src/components/user_settings/EditForm.js:49]: I'm worried about what this is doing. Looks like it'll break.

File: mission-control-client/src/components/user_settings/EditForm.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+})(FormShape)
+
+const mapStateToProps = state => {
+    return{
+        ...state
+    }
+}
+
+
+export default connect(mapStateToProps, {})(EditForm)
```
Comment (dionys93) [inline mission-control-client/src/components/user_settings/EditForm.js:64]: Change empty object to `null` to match with convention

Comment (Artyk77) [reply mission-control-client/src/components/user_settings/EditForm.js]: cool thx

File: mission-control-client/src/components/auth/Login.js [hunk 4]
```diff
@@ -100,12 +101,21 @@ export default withFormik({
     axios
       .post(URL, packet)
       .then(res => {
+        console.log(res)
```
Comment (dionys93) [inline mission-control-client/src/components/auth/Login.js:104]: Remove console.log

File: mission-control-client/src/actions/settingsActions.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+            .catch( err => dispatch({type: GET_MC_ROLES_ERROR, payload: err.response}) )
+    }
+}
+
+export const UPDATE_MC_ROLES_START = 'UPDATE_MC_ROLES_START'
+export const UPDATE_MC_ROLES_SUCCESS = 'UPDATE_MC_ROLES_SUCCESS'
+export const UPDATE_MC_ROLES_ERROR = 'UPDATE_MC_ROLES_ERROR'
+
+export const updateMCRoles = (values) => {
+    console.log(values)
```
Comment (dionys93) [inline mission-control-client/src/actions/settingsActions.js:36]: Remove  console.log

File: mission-control-client/src/components/auth/Login.js [hunk 6]
```diff
@@ -100,12 +101,21 @@ export default withFormik({
...
   }
 })(FormShape);
+
+const mapStateToProps = state => {
+  return {
+    ...state
+  }
+}
+
+export default connect(mapStateToProps, {})(FormikLogin)
```
Comment (dionys93) [inline mission-control-client/src/components/auth/Login.js:121]: Change empty object to `null` to match with `react-redux` convention

File: mission-control-client/src/components/layout/Avatar.js [hunk 7]
```diff
@@ -44,4 +63,9 @@ const AvatarMenu = () => {
   );
 };
 
+const mapStateToProps = state => {
```
Comment (dionys93) [inline mission-control-client/src/components/layout/Avatar.js:66]: I believe this still needs to be connected in `connect`

File: mission-control-client/src/components/user_settings/EditForm.js [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+            password: password || '',
+            newpassword: newpassword || '',
+            confirmpassword: confirmpassword || ''
+        }    
+    },
+    validationSchema: Yup.object().shape({
+        // password: Yup.string().required()
+    }),
+    handleSubmit(values, props){
+        console.log(values, props)
```
Comment (dionys93) [inline mission-control-client/src/components/user_settings/EditForm.js:52]: Remove console.log

File: mission-control-client/src/components/user_settings/EditProfile.js [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+  );
+}
+
+const mapStateToProps = state => {
+    return {
+        ...state
+    }
+}
+
+export default connect(mapStateToProps, {})(Edit)
```
Comment (dionys93) [inline mission-control-client/src/components/user_settings/EditProfile.js:28]: Change empty object to `null`

File: mission-control-client/src/reducers/settingsStore.js [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+                ...state,
+                isLoading:true
+            }
+        case UPDATE_MC_ROLES_SUCCESS:
+            return{
+                ...state,
+                isLoading:false,
+                missionControlUsers: state.missionControlUsers.map( user => {
+                        if(user.userId === action.payload.id){
+                            console.log('here')
```
Comment (dionys93) [inline mission-control-client/src/reducers/settingsStore.js:68]: Toss out console.log

**Review by dionys93 — CHANGES_REQUESTED** [review]
See comments

**Pull request closed without merging**